webui/react/src/pages/ExperimentList.tsx

```diff
@@ -140,14 +140,6 @@ const ExperimentList: React.FC<Props> = ({ project }) => {
   const { settings, updateSettings, resetSettings, activeSettings } =
     useSettings<ExperimentListSettings>(settingsConfig);
 
-  const tableOffset = (() => {
-    if (settings.tableOffset > total) {
-      const newTotal = settings.tableOffset > total ? total : total - 1;
-      return settings.tableLimit * Math.floor(newTotal / settings.tableLimit);
-    }
-    return settings.tableOffset;
```

Comment:
if i understand it correctly, we don't need this logic. even if offset is bigger than total, API just return empty array with pagination info.

Reply:
not sure where where this came from, but I verified that `offset > total` does not cause any issues (i have seen it cause errors elsewhere)